src/app/cart/cart.component.html

```diff
@@ -0,0 +1,66 @@
+<div class="modal" id="myModal">
+    <div class="modal-dialog">
+        <div class="modal-content">
+
+            <div class="modal-header">
+                <h4 class="modal-title">My Cart
+                    <span *ngIf="groceryCart.length >0">({{groceryCart.length}}) item</span>
+                    <span *ngIf="!groceryCart.length">0</span>
+                </h4>
+                <button type="button" class="close" data-dismiss="modal" (click)="resetCart()">&times;</button>
+            </div>
+
+            <div class="modal-body" *ngIf="!groceryCart.length; else displayItems">
+                <div class="cart-body">
+                    <h3>No items in your cart</h3>
+                    <p>You favourite items are just a click away</p>
+                </div>
+            </div>
+
+            <ng-template #displayItems>
+                <div class="card mb-3">
+                    <div class="row no-gutters" *ngFor="let product of groceryCart">
+                        <div class="col-md-2">
+                            <img src="assets/{{product.imageURL}}" class="card-img" alt="{{product.name}}"
+                                title="{{product.name}}">
+                        </div>
+                        <div class="col-md-10" style="margin-left: -100px;">
+                            <div class="card-body" style="text-align: center;">
+                                <h5 class="card-title">{{groceryCart.name}}</h5>
+                                <div>
+                                    <i class="fa fa-plus-square icon-fa" (click)="addProductQuantity(product)" aria-label="increase product unit"></i>
+                                    <span>{{product.count}}</span>
+                                    <i class="fa fa-minus-square icon-fa" (click)="removeProduct(product)" aria-label="decrease product unit"></i>
+                                    <span class="item-price">X &nbsp; &nbsp;Rs. {{product.price}}</span>
```

Comment:
IMHO `addProductToCart` and `removeProductFromCart` could be better names for the above functions